Resources/config/schema/mongodb-1.0.xsd

```diff
@@ -68,8 +68,19 @@
     <xsd:attribute name="readPreference" type="read-preference" />
     <xsd:attribute name="replicaSet" type="xsd:string" />
     <xsd:attribute name="ssl" type="xsd:boolean" />
+    <xsd:attribute name="tls" type="xsd:boolean" />
+    <xsd:attribute name="tlsAllowInvalidCertificates" type="xsd:boolean" />
+    <xsd:attribute name="tlsAllowInvalidHostnames" type="xsd:boolean" />
+    <xsd:attribute name="tlsCAFile" type="xsd:string" />
+    <xsd:attribute name="tlsCertificateKeyFile" type="xsd:string" />
+    <xsd:attribute name="tlsCertificateKeyFilePassword" type="xsd:string" />
+    <xsd:attribute name="tlsDisableCertificateRevocationCheck" type="xsd:boolean" />
+    <xsd:attribute name="tlsDisableOCSPEndpointCheck" type="xsd:boolean" />
+    <xsd:attribute name="tlsInsecure" type="xsd:boolean" />
     <xsd:attribute name="socketTimeoutMS" type="xsd:integer" />
     <xsd:attribute name="username" type="xsd:string" />
+    <xsd:attribute name="retryReads" type="xsd:string" />
+    <xsd:attribute name="retryWrites" type="xsd:string" />
```

Comment:
```suggestion
    <xsd:attribute name="retryReads" type="xsd:boolean" />
    <xsd:attribute name="retryWrites" type="xsd:boolean" />
```

Reply:
Sorry, that was sloppy. Fixed and squashed